src/QuartoNotebookWorker/test/runtests.jl

```diff
@@ -0,0 +1,5 @@
+using Test, QuartoNotebookWorker
+
+@testset "QuartoNotebookWorker" begin
+    @test QuartoNotebookWorker.Packages.is_precompiling() == false
```

Comment:
explain here what it means that `is_precompiling` is false, why it makes sense to test that